src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets

```diff
@@ -0,0 +1,39 @@
+<!--
+***********************************************************************************************
+Sdk.DefaultItems.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="$(MSBuildThisFileDirectory)BundledVersions.props"
+		Condition="Exists('$(MSBuildThisFileDirectory)BundledVersions.props')" />
+
+  <Choose>
+    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(_TargetFrameworkVersionWithoutV)' == '2.1'" >
+      <PropertyGroup>
+        <AspNetCoreSharedFrameworkPackageName Condition="'$(AspNetCoreSharedFrameworkPackageName)' == ''">Microsoft.AspNetCore.App</AspNetCoreSharedFrameworkPackageName>
+        <AspNetCoreSharedFrameworkVersion Condition="'$(AspNetCoreSharedFrameworkVersion)' == ''">$(BundledMicrosoftAspNetCoreAppPackageVersion)</AspNetCoreSharedFrameworkVersion>
```

Comment:
For consistency with `RuntimeFrameworkVersion`, which is used for NETCore.App, how about `AspNetCoreRuntimeFrameworkVersion`?